internal/kldrest/receiptstore.go

```diff
@@ -194,7 +194,7 @@ func (r *receiptStore) getReplies(res http.ResponseWriter, req *http.Request, pa
 	}
 
 	// Call the persistence tier - which must return an empty array when no results (not an error)
-	results, err := r.persistence.GetReceipts(skip, limit, ids)
+	results, err := r.persistence.GetReceipts(skip, limit, ids, req.FormValue("since"), req.FormValue("from"), req.FormValue("to"))
```

Comment:
Not a big deal, but maybe better to move the input validation of `since` from `GetReceipts()` to here (since the other fields are validated here)

Reply:
I'm ok for us to do it in a subsequent change

Reply:
Thanks @vdamle - making this change. It's still a little clunky, because memory receipts don't have the filtering implemented (didn't seem worth it in the cost vs. benefit)